package.json

```diff
@@ -39,7 +39,7 @@
     "styled-components": "^4.3.2",
     "styled-icons": "^8.1.0",
     "typesafe-actions": "^4.4.2",
-    "typescript": "~3.5.2",
+    "typescript": "~3.9.6",
```

Comment:
so we are going back to `3` or is this because of rebasing?

Reply:
node issues with 4.1 version, will deal with node in a separate PR

Reply:
should then the `node` be updated before the final version of this or?

Reply:
3.9.6 has no issues while running checks locally, Tom wanted to have at least some upgrade so we're merging 3.9.6 while I update node in another PR (there's a card for it since a while ago), and then upgrade typescript again, trying to upgrade both and match dependencies in this PR wasn't working out.

Reply:
cool